assets/app/view/game/buy_sell_shares.rb

```diff
@@ -16,7 +16,7 @@ def render
         current_entity = step.current_entity
 
         ipo_share = @corporation.shares[0]
-        pool_share = @game.share_pool.shares_by_corporation[@corporation][0]
+        pool_shares = @game.share_pool.shares_by_corporation[@corporation].group_by(&:percent).values.collect(&:first)
```

Comment:
I suggest using map instead of collect as that seems to be the pattern in the codebase.

Reply:
agreed, i can make this change locally